pkg/plugin/test-keeper/plugin/event_handler_gh_test.go

```diff
@@ -235,7 +230,7 @@ var _ = Describe("Test Keeper Plugin features", func() {
 
 			gock.New("https://api.github.com").
 				Post("/repos/" + repositoryName + "/statuses").
-				SetMatcher(ExpectPayload(toHaveFailureState)).
+				SetMatcher(ExpectPayload(toBe(github.StatusFailure, "No tests in this PR :("))).
```

Comment:
would make sense to use constants for these strings?

Reply:
Sure, good idea. On the way

Reply:
def5e6c

Reply:
def5e6c